plugin/lsp.vim

```diff
@@ -78,7 +78,7 @@ augroup END
 
 # LSP commands
 command! -nargs=? -bar -range LspCodeAction lsp.CodeAction(<line1>, <line2>, <q-args>)
-command! -nargs=0 -bar LspDiagCurrent lsp.LspShowCurrentDiag()
+command! -nargs=0 -bang -bar LspDiagCurrent lsp.LspShowCurrentDiag(<bang>false)
```

Comment:
I think that it would be more intuitive if these wore changed, so that `:LspDiagCurrent` would only show diags directly under the cursor, while `:LspDiagCurrent!` should scan forward for one.  But it's also worth being backward compatible.

Reply:
I think it'd be more intuitive too ;)